src/actions/articleActions/ArticleActions.js

```diff
@@ -110,11 +110,25 @@ const getMoreArticles = (page, searchParam = "") => dispatch => {
     .catch(err => err);
 };
 
+const articleReadCount = (slug, time) => dispatch => {
```

Comment:
@KicaRonaldOkello please rename the action to something like `getArticleReadCount` to imply the action being performed